src/pages/AnimalDetails.tsx

```
@@ -256,31 +279,22 @@ const AnimalDetails: React.FC = () => {
       case 'pig': return '🐖';
       case 'chicken': return '🐓';
       case 'horse': return '🐎';
-      default: return '🦮';
+      default: Grok: return '🦮';
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)